Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/npc.yml

```diff
@@ -0,0 +1,10 @@
+- type: entity
+  id: CrateNPCFox
+  parent: CrateLivestock
+  name: Fox crate
+  description: "A crate containing a single fox."
+  components:
+  - type: StorageFill
+    contents:
+      - id: MobFox
+        amount: 1
```

Comment:
just one?

Reply:
same principle as the corgi and cat crate. One pet for 1k

Reply:
buuut I can make it more